src/Models/HydrostaticFreeSurfaceModels/matrix_implicit_free_surface_solver.jl

```diff
@@ -104,7 +104,7 @@ end
     Az   = Azᶜᶜᶜ(i, j, 1, grid)
```

Comment:
@simone-silvestri, should this be `Az   = Azᶜᶜᶜ(i, j, Nz, grid)`

Reply:
good catch